frontend/packages/dev-console/src/components/helm/HelmReleasesDetailsPage.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { match as RMatch } from 'react-router';
+import * as _ from 'lodash';
+import { Firehose } from '@console/internal/components/utils';
+import { NamespaceBar } from '@console/internal/components/namespace';
+import HelmReleaseDetailsPage from './HelmReleaseDetailsPage';
+
+export interface HelmReleasesDetailsPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+
+const HelmReleasesDetailsPage: React.FC<HelmReleasesDetailsPageProps> = ({ match }) => {
```

Comment:
Why is Helm releases plural in the name? This is a details page for a single helm release.

Reply:
Changed this file name to `HelmReleaseDetails`